setup/artifacts/s3.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: single
+title:  "Configure S3 Artifact"
+sidebar:
+  nav: setup
+---
+
+{% include toc %}
+
+Spinnaker stages that read data from artifacts can read S3 files directly.
+
+Several settings can be configured.
```

Comment:
I think you can delete this sentence, unless you're going to enumerate the several setting here.